modules/doc/content/framework_development/git.md

```diff
@@ -20,13 +20,15 @@ however, you might find yourself with a huge number of remotes that you don't re
 
 ## Git completion
 
-To enable bash completion for git (so you can tab-complete 'git co' or 'git br' commands) download [git_completion.sh](http://mooseframework.org/static/media/uploads/files/git_completion.sh) to your home directory, and append the following line to the end of your ~/.bash_profile (or ~/.bashrc)
+#### Using the bash shell
+
+To enable bash completion for git (so you can tab-complete 'git co' or 'git br' commands) download [git_completion.sh](https://github.com/git/git/blob/master/contrib/completion/git-completion.bash) to your home directory, and append the following line to the end of your ~/.bash_profile (or ~/.bashrc)
```

Comment:
```suggestion
To enable bash completion for git (so you can tab-complete 'git co' or 'git br' commands) download [git_completion.bash](https://github.com/git/git/blob/master/contrib/completion/git-completion.bash) to your home directory, and append the following line to the end of your ~/.bash_profile (or ~/.bashrc)
```

If the name of the script changed, we should be consistent in the documentation.